script/Arcueid/Day 9/09_00_ARC09_2B.txt

```diff
@@ -13,8 +13,9 @@ I did not notice that anything was amiss...
 [sha:5222c64e28982a05b1dd1f60fa99513d7ff7417d]{
 -- Page 10005, Offset 21410.
 -- 「翡翠は悪くないよ。いつもの事だし、そう大げさにとらない、とらない」
+// Three clauses in one sentence either needs to be split like this or use a semicolon
```

Comment:
minor nit, but don't leave comments like this in script files. Without the context of the github pr diff, it doesn't really mean anything. Please remove before you merge